### ISSUE TITLE
"Abnormality of cerebral white matter" subclass of "Abnormal CNS myelination"

### ISSUE THREAD

**Issue by drseb** [obophenotype/human-phenotype-ontology#3304]
Sitting here with @sheridae and discussing if "Abnormality of cerebral white matter" should be a subclass of "Abnormal myelination"? He has discussed with several other experts and they were wondering why this link is not present. 

@pnrobinson @mellybelly @nicolevasilevsky can you comment?

**Comment by pnrobinson** [comment]
Abnormal myelination is a subclass of Abnormality of nervous system morphology
and
Abnormality of the cerebral white matter is a subclass of Abnormality of the cerebral white matter

I am not seeing the relation Abnormality of cerebral white matter" subclass_of "Abnormal myelination" (Which I do not think would be correct). ??

**Comment by drseb** [comment]
I think the argument was that white matter is basically just myelin... to cite wikipedia: "White matter refers to areas of the central nervous system (CNS) that are mainly made up of myelinated axons". (I am not an expert here...)

The real experts were expecting all of the terms of the white matter subtree to be grouped under abnormal CNS myelination. Is this really wrong? Thanks for comments.

**Comment by drseb** [comment]
Or let me rephrase: they expected that abnormalities of white matter always refer to abnormalities in the myelination (of CNS).

**Comment by drseb** [comment]
@sheridae is this kind of what they expected?

**Comment by pnrobinson** [comment]
There is myelin in the peripheral nervous system as well. 
The white matter is a part of the brain that is made up of many components including myelin, and many nerves outside of the whilte matter also have myelin. I do not think that this assertion is (always) correct!

**Comment by drseb** [comment]
I have the feeling things are a bit mixed up now. The suggestion does not mean that all myelin is only in the white matter. 
However, I think I have a solution:
We add a new subclass "Abnormal myelination of cerebral white matter" and put all myelination-abnormalities of the white matter there. "Abnormal myelination of cerebral white matter" will then be a subclass of "Abnormality of cerebral white matter" and "Abnormal CNS myelination". Does that sound good @pnrobinson @sheridae ?

**Comment by pnrobinson** [comment]
I do not understand the need for this class right now. What exactly cannot be done with the current classes? Is there a diseases that has this distinction? In general, we do not want to have a class that has a single child, so are there any other children of Abnormal CNS myelination if we do it this way?

**Comment by drseb** [comment]
>  In general, we do not want to have a class that has a single child, so are there any other children of Abnormal CNS myelination if we do it this way?

There are already >10 subclasses of "Abnormal CNS myelination"

> I do not understand the need for this class right now. What exactly cannot be done with the current classes?

I will get back to you on this later.

**Comment by pnrobinson** [comment]
Sorry -- maybe I misunderstood. Do you mean 
"Abnormal myelination of cerebral white matter" subclass_of "Abnormal CNS myelination"
or
"Abnormal CNS myelination" subclass_of "Abnormal myelination of cerebral white matter"
But in any case please provide feedback about what diseases are to be annotated

**Comment by drseb** [comment]
We now understand what is going on. We will move this to guidelines for annotating patients. Sorry for the confusion. I'll close here
